src/AppBundle/Controller/TaskController.php

```diff
@@ -29,6 +30,8 @@ public function createAction(Request $request)
         $form->handleRequest($request);
 
         if ($form->isValid()) {
+            $author = $this->getUser();
+            $task->setAuthor($author);
```

Comment:
`$task->setAuthor($this->getUser());`? Why do you need to assign an intermediate variable (`$author`)